plugins/bufbuild/validate-java/v1.1.0/buf.plugin.yaml

```
@@ -0,0 +1,16 @@
+version: v1
+name: buf.build/bufbuild/validate-java
+plugin_version: v1.1.0
+source_url: https://github.com/bufbuild/protoc-gen-validate
+integration_guide_url: https://github.com/bufbuild/protoc-gen-validate/blob/main/docs.md#java
+description: Generates Java code to validate Protobuf messages using protoc-gen-validate constraints.
+deps:
+  - plugin: buf.build/protocolbuffers/java:v25.2
```

```suggestion
  - plugin: buf.build/protocolbuffers/java:v27.3
```

Not sure if this is compatible yet with the latest Java artifact or if it needs to stay on 25.x. cc @rodaine

---

Noticed the dependency on protobuf-java hasn't updated - going to stay on 25.x for now.